### ISSUE TITLE
Question: Possibility of getting VAC ban by using Hammer++?

### ISSUE THREAD

**Issue by rwynx** [ficool2/HammerPlusPlus-Website#27]
Hey, I really liked its new features such as lighting preview and stuff. But I have a question, when I compile the map and trigger the game automatically, is there any possibility of getting banned? (I know the SDK itself isn't a problem, but I was worried about CS:GO)

**Comment by hexaflexahexagon** [comment]
No, that compile option just launches the game normally using some special launch parameters. There's no risk in h++ or any other version of hammer.

**Comment by TheBlokker** [comment]
With the parameter -dev, -insecure you are disable vac and start cargo without vac
